dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/merge/orgunit/OrgUnitMergeValidator.java

```diff
@@ -67,7 +67,7 @@ public void validate( OrgUnitMergeRequest request )
      */
     public ErrorMessage validateForErrorMessage( OrgUnitMergeRequest request )
     {
-        if ( request.getSources().isEmpty() )
+        if ( request.getSources().size() < 2 )
```

Comment:
Are more than 2 supported?

Reply:
Yes, any number, but _at least_ 2 is required.